web/GestureHandler.js

```diff
@@ -147,10 +147,7 @@ class GestureHandler {
   }
 
   sendEvent = nativeEvent => {
-    const {
-      onGestureHandlerStateChange: onHandlerStateChange,
-      onGestureHandlerEvent: onGestureEvent,
-    } = this.ref.props;
+    const { onHandlerStateChange, onGestureEvent } = this._propsRef.current;
```

Comment:
Why did you remove destructuring here? `onGestureHandlerStateChange` and `onHandlerStateChange` (note *Gesture* part) aren't interchangeable.

```
 const { onGestureHandlerStateChange: onHandlerStateChange,	
         onGestureHandlerEvent:       onGestureEvent, } = (...)
```

Reply:
Because with take them directly from props and they defined with `onHandlerStateChange, onGestureEvent` in props.

Reply:
I'm afraid I don't understand, could you elaborate?

My concern is, we set those two properties in [`React.cloneElement`](https://github.com/software-mansion/react-native-gesture-handler/blob/7564162fe6bf4ddb04f2ce18f8dfd4dfe905c98c/createHandler.js#L347-L348) (this is used to nest handlers). I may be wrong, but it won't ignore those?

Have you tested your PR against nested handlers? If it works then we're okay.

Reply:
You are totally right I understand that not, because this reanimated useAnimatedGestureHandler don't work with this PR.